app/scripts/lib/setupSentry.js

```diff
@@ -93,11 +94,23 @@ function getClientOptions() {
     // we can safely turn them off by setting the `sendClientReports` option to
     // `false`.
     sendClientReports: false,
-    tracesSampleRate: 0.01,
+    tracesSampleRate: 1.0,
     transport: makeTransport,
   };
 }
 
+function setCircleCiTags() {
+  Sentry.setTag('circleci.enabled', !!process.env.CIRCLECI);
+
+  if (process.env.CIRCLECI) {
+    Sentry.setTag('circleci.branch', process.env.CIRCLE_BRANCH);
+    Sentry.setTag('circleci.buildNum', process.env.CIRCLE_BUILD_NUM);
+    Sentry.setTag('circleci.job', process.env.CIRCLE_JOB);
+    Sentry.setTag('circleci.nodeIndex', process.env.CIRCLE_NODE_INDEX);
+    Sentry.setTag('circleci.prNumber', process.env.CIRCLE_PR_NUMBER);
+  }
+}
```

Comment:
@davidmurdoch this is all getting set at Extension build time, not at E2E test run time.  So for instance, `circleci.job` will be `prep-build-test-webpack` instead of `test-e2e-chrome-webpack`.   What's the best way to evaluate this at E2E test run time?

Reply:
webpack uses the build.yml to load env vars into the build process, just like the gulp build does, so it should Just Work™️ .